CHANGELOG.md

```diff
@@ -10,6 +10,30 @@ For unreleased changes see [WHATSNEW.md](WHATSNEW.md)
 
 # [Released]
 
+## [2.1.0] - 2024-03-05
+
+### Added
+
+- Option `--raw` for `fmg get devices`
+- Support for HTTP `PATCH` and `DELETE` method in FortiClientEMS
+- Option `--smtp` for `fgt config check`
+
+
+### Changed
+
+- `fmg get devices` also shows ha nodes if device is a cluster
+- Make `Fortinet.api()` more generic to support more methods
```

Comment:
Perhaps be more specific here: `HTTP methods`

Reply:
The complete sentence is:

- Support for HTTP `PATCH` and `DELETE` **method** in FortiClientEMS

So, the word "method" is already present after the methods.

Reply:
Hmm I would nonetheless add the `HTTP` word in line 25, because this is to enable what then has been added.

Reply:
Yes, of course you're right. I looked at the wrong line.
Has been added now.